.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__/
```

Comment:
Git will not try to check these files in now.  This is desirable, as they are just cruft python leaves behind to make itself faster on subsequent invocations.